moped-editor/src/utils/mapHelpers.js

```diff
@@ -32,57 +62,114 @@ export const getGeoJSON = e =>
     type: e.features[0].type,
   };
 
+/**
+ * Determine if a feature is present/absent from the feature collection state
+ * @param {Object} selectedFeature - Feature selected
+ * @param {Array} features - Array of GeoJSON features
+ * @return {Boolean} Is feature present in features of feature collection in state
+ */
 export const isFeaturePresent = (selectedFeature, features) =>
   features.some(feature => isEqual(selectedFeature, feature));
 
-export const createProjectLayerConfig = (
-  polygonId,
-  layerSourceName,
+/**
+ * Create a configuration to set the Mapbox spec styles for selected/unselected/hovered layer features
```

Comment:
nice!!! love all these docstrings! Ignore what I said in my last PR review. You're way ahead of me on this!